src/read_agent_logging.in

```diff
@@ -20,22 +20,32 @@ URL="${1:-http://localhost:24231/metrics}"
 INTERVAL=${COLLECTD_INTERVAL:-10}
 
 # The part of the input that sed looks at looks like:
-#   stackdriver_successful_requests_count{grpc="false"} 255
-#   stackdriver_failed_requests_count{grpc="false",code="400"} 14
+# 1. If the Logging Agent uses REST.
+#   stackdriver_successful_requests_count{grpc="false",code="200"} 255
```

Comment:
If you were planning on combining metrics, shouldn't you be changing the sample inputs?